packages/linea-ens-app/src/components/@molecules/TransactionDialogManager/stage/TransactionStageModal.tsx

```diff
@@ -506,6 +507,18 @@ export const TransactionStageModal = ({
           </Outlink>
         )}
         {lowerError && <Helper type="error">{lowerError}</Helper>}
+        {stage === 'complete' ? (
+          <Content title="" hideHeading={true}>
+            {{
+              warning: {
+                type: 'warning',
+                message:
+                  'You will be able to use your new domain anywhere ENS is supported once the transaction is finalized on L1. This process takes 8 to 32 hours. <a href="https://docs.linea.build/" style="color:blue;" target="_blank">Read more</a>',
```

Comment:
Can you use `useTranslation` and put the texts in the json files in `public/locales/en/`